root/edit/index.tt

```diff
@@ -1,8 +1,11 @@
 [%- WRAPPER 'layout.tt' title=l('Edit #{id}', { id => edit.id }) -%]
     <div id="content">
 
-        [% INCLUDE 'edit/edit_header.tt' %]
         [%- edit_json_obj = React.to_json_object(edit) -%]
+        [%- edit_json_obj_no_data = edit_json_obj -%]
```

Comment:
Not sure this works (i.e. makes a copy) actually -- I'm getting `TypeError: Cannot read property 'relationship' of undefined` on /edit/$id pages.

Unless I'm missing some other obvious way, we could probably use http://template-toolkit.org/docs/manual/VMethods.html#section_import by creating an empty object and "importing" the other object into it.